src/build_platform/local_docker.rs

```diff
@@ -133,27 +115,25 @@ impl LocalDocker {
             }
         };
 
-        let mut docker_args = if env_var_args.is_empty() {
-            docker_args
-        } else {
-            let mut build_args = vec![];
-
-            env_var_args.iter().for_each(|arg_value| {
-                build_args.push("--build-arg");
-                build_args.push(arg_value.as_str());
-            });
-
-            docker_args.extend(build_args);
-            docker_args
-        };
-
-        docker_args.push(into_dir_docker_style);
-
-        // docker build
-        let mut cmd = QoveryCommand::new("docker", &docker_args, &self.get_docker_host_envs());
-
-        let exit_status = cmd.exec_with_abort(
-            Duration::minutes(BUILD_DURATION_TIMEOUT_MIN),
+        // FIXME: pass a Vec<(key, value)> instead of spliting always the string
```

Comment:
Did you forget to fix?

Reply:
I haven't finished yet :P wanted to check the test first
Going to do it